Source/HelixToolkit.Wpf.SharpDX/Controls/DPFCanvas.cs

```diff
@@ -132,7 +133,26 @@ public IRenderer Renderable
         /// </summary>
         Device IRenderHost.Device
         {
-            get { return this.device; }
+            get { return device; }
+        }
+
+        public static readonly DependencyProperty EffectsManagerProperty =

```

Comment:
We add two new `DependencyProperty` on `DPFCanvas`. This allows a third-party to provide an implementation of `IRenderTechniquesManager` and `IEffectsManager`.